map/framework.cpp

```diff
@@ -441,6 +443,8 @@ Framework::Framework(FrameworkParams const & params)
   {
     m_localAdsManager.Startup();
     m_localAdsManager.GetStatistics().SetUserId(GetPlatform().UniqueClientId());
+    local_ads::IconsInfo::Instance().SetSourceFile(
```

Comment:
Again, doing IO on the UI thread is not a good idea. It does not accelerate startup of our application.